### PR TITLE
chore: simplify streaming on/off logic

### PR DESCRIPTION
**Pull request by ilyam8**
Replacing if/else branch in templates with the ternary function in values.

### CODE REVIEW

**Comment by ilyam8** [comment]
@M4itee can you please review this one?

File: charts/netdata/values.yaml [hunk 1]
```diff
@@ -284,7 +284,7 @@ child:
       path: /etc/netdata/stream.conf
       data: |
         [stream]
-          enabled = yes
+          enabled = {{ ternary "yes" "no" .Values.parent.enabled }}
```
Comment (M4itee) [inline charts/netdata/values.yaml:287]: I never used that kind of function in helm, tho I love it in terraform. I learned something new, thx :D

**Review by M4itee — APPROVED** [review]

File: charts/netdata/values.yaml (hunk 1 above)
Comment (ilyam8) [reply charts/netdata/values.yaml:287]: Helm extends go template lang with [sprig functions](http://masterminds.github.io/sprig/).

**Comment by ilyam8** [comment]
@witalisoft @Ferroin can I please get 2nd review?

**Review by witalisoft — APPROVED** [review]

**Pull request merged**